cmd/emailreport/README.md

```diff
@@ -10,7 +10,26 @@ file can be specified by command line or it is installed in
 WARNING: The password for the SMTP authentication will be in TOML
 configuration text plane file.
 
-# Usage
+## Options
+`emailreport` takes a number of options.
+
+### TOML configuration file path
```

Comment:
Is this description correct? it should refer to L19, right?

Reply:
Ops!, thanks @chavafg